pulseguardian/guardian.py

```diff
@@ -141,7 +141,11 @@ def update_queue_information(self, queue_data, all_bindings):
                     logging.info(
                         "Queue '{0}' owner, {1}, isn't in the db. Creating "
                         "the user.".format(q_name, owner_name))
-                    owner = PulseUser.new_user(owner_name)
+                    # PulseUser needs at least one owner as well, but since
+                    # we have no way of knowing who really owns it, find the
+                    # first admin, and set it to that.
+                    user = User.query.filter(User.admin == 1).first()
```

Comment:
I think this should be True and not 1, as the field type for admin is Boolean. It's just implemented as an integer in sqlite.